src/layers/medPython/cmake/embed_python.cmake

```diff
@@ -13,9 +13,15 @@
 #
 ################################################################################
 
-function(embed_python target)
```

Comment:
This file was adapted to compile, but it will change significantly in the next PR.